pkg/kubelet/certificate/bootstrap/bootstrap_test.go

```diff
@@ -320,6 +320,7 @@ users:
 	if err != nil {
 		t.Fatal(err)
 	}
+	f.Close()
```

Comment:
You don't need a defer here?

Reply:
Yes.Because the fd is not used in the context after that,  no need defer when we know clearly what we are doing.

Reply:
I don't understand this response. The file is used below

Reply:
Well...we just use the f.Name() below, we don't use fd anymore.It has no affacting when we call f.Name after call f.Close().
Sorry That to avoid ambiguity I Will refactor this pr to make it more clear.